src/client/components/HealthCareApp.jsx

```diff
@@ -109,18 +107,30 @@ class HealthCareApp extends React.Component {
       if (!response.ok) {
         throw new Error(response.statusText);
       }
-      store.dispatch(updateSubmissionStatus('submitSucceeded', response.json()));
+      response.json().then(data => {
+        this.props.onUpdateSubmissionStatus('applicationSubmitted', data);
+        console.log('data', data);
+        this.props.onUpdateSubmissionId(data.formSubmissionId);
+        this.props.onUpdateSubmissionTimestamp(data.timeStamp);
+      });
+      setTimeout(() => {
+        this.context.router.push(this.getUrl('next'));
+        this.scrollToTop();
+      }, 8000);
     }).catch(error => {
-      store.dispatch(updateSubmissionStatus('submitFailed', error));
+      setTimeout(() => {
+        this.props.onUpdateSubmissionStatus('submitFailed', error);
+      }, 5000);
```

Comment:
Agreed! But right now we don't get anything useful back from the endpoint about why the submission didn't go through so I have nothing to show the user at this point. We could add some place holder text for now and just say "Please review your application and make sure that you have completed ALL required field before trying to resubmit."